src/components/questions_answers/QnA_Components/AddAnswer.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export default function AddAnswer({ question, setShowModal }) {
+  // const [question, setQuestion] = useState(question);
+  const [answer, setAnswer] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [email, setEmail] = useState('');
+  // const [photos, setPhotos] = useState([]);
+  // const [photosToUpload, setPhotosToUpload] = useState([]);
+
+  function handleSubmit() {
+    if (answer.length && nickname.length && email.length) {
+      setShowModal(false);
+    }
+  }
+
+  //   function handlePhotos(e) {
+  //     setPhotosToUpload(e.target.files);
+  //     handleUpload(e.target.files);
+  //   }
+
+  // function handleUpload(files) {
+  //     if (files.length <= 5) {
+  //       const uploads = [];
+  //       for (let i = 0; i < files.length; i++) {
+
+  //       }
+  //       setPhotos(uploads);
+  //     }
+  //   }
+
+  // function handleClick(e) {
+  //   if (e.target !== e.currentTarget) {
+  //     e.stopPropagation();
+  //     return;
+  //   }
+  //   setShowModal(false);
+  // }
+
+  return (
+    <div>
+      <div>
+        <div>
+          <div>
+            <h5>Submit Your Answer: {question.question_body}</h5>
+            <button
+              type="button"
+              onClick={() => setShowModal(false)}
+              data-dismiss="modal"
+              aria-label="Close"
+            />
```

Comment:
This button seems to be doing the same thing as the button on lines 100-106. Can this be removed?